check_monit.py

```diff
@@ -130,7 +135,7 @@ def main(args):
 
     for service in services:
         monitor = int(service.find('monitor').text)
-        if monitor == 1:
+        if monitor == 1 or monitor == 2:
```

Comment:
```suggestion
        if monitor in (1, 2):
```

Reply:
Nice, thanks!

Do you think this should be published without an answer about the monit documentation for xml?